### DISCUSSION TITLE
List of unexplained words and concepts

### DISCUSSION THREAD

**Discussion by marmolotte** [FachschaftMathPhysInfo/Programmiervorkurs#149] [Ideas]
Some words in the script seem to be seen as common knowledge and left unexplained, which does not make it very helpful as a ground zero starting point. Please include definitions of those in the glossary

**Comment by marmolotte** [comment]
-Terminal (Lektion 0, „öffnet ein Terminal“, ja ich weiß, vielleicht fühlen sich manche veräppelt, wenn das dann nochmal erklärt wird, aber ich hätte damals nach dem Satz nicht gewusst, was ich machen soll)
-Anwendung („wann immer ihr über die Anwendungen ein Terminal startet […]“, Lektion 2)
-

**Comment by fschledorn** [comment]
> Some words in the script seem to be seen as common knowledge and left unexplained, which does not make it very helpful as a ground zero starting point. Please include definitions of those in the glossary

Shouldn’t this be combined with the glossary idea from Tom Rix (see #93  and #101) and the Issue #150 or is there a particular reason for opening both a Diskussion and an Issue? All of these Ideas could be effectively solved by implementing a good glossary where words and concepts are explained more in detail. Tom provided a framework a few years ago that we could take a look at. Dont know how far he got exactly but it is my understanding that he finished the framework for it.

**Reply by marmolotte** [reply]
Yes, apparently I forgot that the glossary exists exactly after opening the discussion
